tests/tests_spdx.py

```diff
@@ -0,0 +1,128 @@
+from __future__ import absolute_import, unicode_literals
+
+from webapp.exceptions import ValidationError
+
+import unittest
+from webapp.spdx import (
+    get_spdx_license_symbols,
+    is_spdx_or_only_expression,
+    validate_spdx_license_expression,
+)
+
+
+class SPDXTestCase(unittest.TestCase):
+
+    def test_validate_expression_with_single_license(self):
+        licenses = 'MIT'
+        expr = validate_spdx_license_expression(licenses)
+        self.assertEqual(expr, 'MIT')
+
+    def test_validate_expression_with_multiple_licenses(self):
+        licenses = 'CC-BY-2.0 OR (GPL-3.0 AND MIT)'
+        expr = validate_spdx_license_expression(licenses)
+        self.assertEqual(expr, 'CC-BY-2.0 OR (GPL-3.0 AND MIT)')
+
+    def test_validate_expression_with_invalid_licenses(self):
+        licenses = 'MIT OR Some-other-license'
+        with self.assertRaises(ValidationError) as ctx:
+            validate_spdx_license_expression(licenses)
+        self.assertEqual(
+            str(ctx.exception),
```

Comment:
was: `ctx.excpection.message` now I just `str` the excpection since message is not valid anymore